python/core/auto_generated/vector/qgsvectorlayer.sip.in

```diff
@@ -1777,7 +1777,7 @@ be updated. This can be used to override default field value expressions.
 .. seealso:: :py:func:`updateFeature`
 %End
 
-    bool changeAttributeValue( QgsFeatureId fid, int field, const QVariant &newValue, const QVariant &oldValue = QVariant(), bool skipDefaultValues = false );
+    bool changeAttributeValue( QgsFeatureId fid, int field, const QVariant &newValue, const QVariant &oldValue = QVariant(), bool skipDefaultValues = false,  QgsExpressionContext *context = 0 );
```

Comment:
Similarly here, could we go with QgsVectorLayerToolsContext instead? It'll be more future proof.